megamek/src/megamek/client/ui/swing/AccessibilityWindow.java

```diff
@@ -0,0 +1,190 @@
+package megamek.client.ui.swing;
+
+import megamek.client.Client;
+import megamek.client.event.BoardViewEvent;
+import megamek.client.ui.IBoardView;
+import megamek.client.ui.swing.boardview.BoardView1;
+import megamek.client.ui.Messages;
+import megamek.common.Coords;
+import megamek.common.Entity;
+import megamek.common.event.*;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.KeyEvent;
+import java.awt.event.InputEvent;
+import java.awt.event.KeyListener;
+import java.util.LinkedList;
+
+public class AccessibilityWindow extends JDialog implements KeyListener {
+
+    public static final int MAX_HISTORY = 10;
+    public static final String ACCESSIBLE_GUI_SHORTCUT = ".";
+
+    Client client;
+    ClientGUI gui;
+    JTextArea chatArea;
+
+    private Coords selectedTarget;
+    private JTextField inputField;
+    private String[] args;
+    public LinkedList<String> history;
```

Comment:
It looks like neither history, nor historyBookmark need to be public, so let's make them private.